pdb.py

```diff
@@ -916,9 +916,8 @@ def _print_lines_pdbpp(self, lines, lineno, print_markers=True):
             # only for the current/marked line really.
             lines = [line.ljust(maxlength) for line in lines]
 
-        if self.config.use_pygments:
-            src = self.format_source('\n'.join(lines))
-            lines = src.splitlines()
+        src = self.format_source('\n'.join(lines))
```

Comment:
`format_source` checks `self.config.use_pygments`

Reply:
This is meant to short-circuit it here, to not unnecessarily split and join.